docs/documentation/maya/ribbon.md

```diff
@@ -110,108 +74,124 @@ Following, we present the key aspects to keep in mind while painting each attrib
-    - Has a range of \[0.0, 2.0\] [^1]
- - **Space Scale Mode** (Enumerator, "Masses + Forces"): Determines if the spatial scaling affects the masses, the forces, or both.
-    - The available options are: Masses, Forces, Masses + Forces.
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
 
 #### Gravity
- - **Gravity** (Float, 1.0): Sets the magnitude of the gravity acceleration.
- - **Gravity Direction** (Float3, {0.0. -1.0, 0.0} ): Sets the direction of the gravity acceleration.
-    - Vectors introduced don't need to be normalized, but they will get normalized internally.
+ - **Gravity** (Float, 0.0): Sets the magnitude of the gravity acceleration.
+    - Has a range of \[0.0, 100.0\]. Upper limit is soft, higher values can be used.
+ - **Gravity Direction** (Float3, {0.0. -1.0, 0.0}): Sets the direction of the gravity acceleration.
+    - Vectors introduced do not need to be normalized, but they will get normalized internally.
 
 ### Advanced Settings
 
 #### Stiffness Settings
  - **Use Custom Stiffness** (Boolean, False): Toggles the use of a custom stiffness value. If enabled, the Material is ignored and the Stiffness parameter is used instead.
-    - If we use a custom stiffness, **Material** and **Stiffness Multiplier** will be disabled and **Stiffness** will be used instead.
+    - If use a custom stiffness is selected, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead.
+ - **Stiffness** (Float, 10^5^): Sets the custom stiffness value.
+    - Its value must be greater than 0.0.
 
 #### Dynamic Properties
  - **Global Damping** (Float, 0.75): Sets the scaling factor applied to the global damping of every point.
-    - Has a range of \[0.0, 1.0\] [^1]
+    - Has a range of \[0.0, 1.0\]. Upper limit is soft, higher values can be used.
  - **Inertia Damper** (Float, 0.0): Sets the linear damping applied to the dynamics of every point.
-    - Has a range of \[0.0, 1.0\] [^1]
+    - Has a range of \[0.0, 1.0\]. Upper limit is soft, higher values can be used.
  - **Rest Length Multiplier** (Float, 1.0): Sets the scaling factor applied to the edge lengths at rest.
-    - Has a range of \[0.0, 2.0\] [^1]
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
  - **Compression Multiplier** (Float, 1.0): Sets the scaling factor applied to the compression resistance of every point.
-    - Has a range of \[0.0, 2.0\] [^1]
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
  - **Stretching Multiplier** (Float, 1.0): Sets the scaling factor applied to the stretching resistance of every point.
-    - Has a range of \[0.0, 2.0\] [^1]
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
  - **Attenuation Velocity factor** (Float, 1.0): Sets the weight of the attenuation applied to the whole simulation driven by the Attenuation Matrix.
```

Comment:
```suggestion
 - **Attenuation Velocity Factor** (Float, 1.0): Sets the weight of the attenuation applied to the whole simulation driven by the Attenuation Matrix.
```
Also, should we add that attenuation applies to the velocities of the simulated vertices instead of the whole simulation?